fili-core/src/main/java/com/yahoo/bard/webservice/async/jobs/stores/HashJobStore.java

```diff
@@ -99,10 +99,10 @@ private boolean satisfiesFilters(Set<JobRowFilter> jobRowFilters, JobRow jobRow)
     private boolean satisfiesFilter(JobRow jobRow, JobRowFilter jobRowFilter) throws IllegalArgumentException {
         JobField filterJobField = jobRowFilter.getJobField();
         FilterOperation filterOperation = jobRowFilter.getOperation();
-        if (!filterOperation.equals(FilterOperation.eq)) {
-            LOG.debug(FILTER_OPERATOR_INVALID.logFormat(filterOperation));
-            throw new IllegalArgumentException(FILTER_OPERATOR_INVALID.format(filterOperation));
-        }
+        Set<String> filterValues = jobRowFilter.getValues();
+
+        String actualValue = jobRow.get(filterJobField);
```

Comment:
Either this should happen after the if-statement (extracting the value stored at a given key before checking if the key exists is a little bit weird), or the if block should check if `actualValue` is `null`. Personally I would recommend the first.